packages/hierarchies/src/test/Utils.ts

```diff
@@ -214,6 +215,23 @@ export function createIModelAccessStub() {
   };
 }
 
+export function createInstanceLabelSelectClauseFactoryStub() {
+  return {
+    async createSelectClause(props: {
+      classAlias: string;
+      className?: string;
+      selectorsConcatenator?: (selectors: TypedValueSelectClauseProps[], checkSelector?: string) => string;
+    }): Promise<string> {
+      return `(
+        SELECT
+         [${props.classAlias}].[CodeValue]
+        FROM [meta].[ECClassDef] AS [c]
+        WHERE [c].[ECInstanceId] =[${props.classAlias}].[ECClassId]
+      )`;
```

Comment:
IMO this should be much simpler e,g. `[${classAlias}].[LabelProperty]`.

Reply:
Simplified to your suggestion